### PR TITLE
Pgbouncer Cluster Dashboard

### PR DESCRIPTION
**Pull request by BominRahmani**
Link to dashboard (https://keithschmitty.grafana.net/d/pgbouncer-cluster-overview/pgbouncer-cluster-overview?orgId=1&refresh=1m)

### CODE REVIEW

File: pgbouncer-mixin/panels.libsonnet [hunk 1]
```diff
@@ -152,5 +154,48 @@ local utils = commonlib.utils;
...
         )
         + g.panel.timeSeries.standardOptions.withUnit('s'),
+
+      alertsPanel:
+        alertList.new('PgBouncer alerts')
+        + alertList.options.UnifiedAlertListOptions.withAlertInstanceLabelFilter(this.grafana.variables.queriesGroupSelectorAdvanced),
+
+      topDatabaseActiveConnection:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top database by active connections',
```
Comment (StefanKurek) [inline pgbouncer-mixin/panels.libsonnet:164]: "Top databases"

File: pgbouncer-mixin/panels.libsonnet [hunk 2]
```diff
@@ -152,5 +154,48 @@ local utils = commonlib.utils;
...
+            Top databases by average duration of queries being processed by PgBouncer.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('s'),
+      topDatabaseNetworkTraffic:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top databases by network traffic',
+          targets=[t.topDatabaseNetworkTrafficReceived, t.topDatabaseNetworkTrafficSent],
+          description=|||
+            Top databases by volume in bytes of network traffic received by PgBouncer.
```
Comment (StefanKurek) [inline pgbouncer-mixin/panels.libsonnet:194]: "...by volume of network traffic received by PgBouncer"

I wouldn't say "bytes" as this unit might change in the graph dynamically

File: pgbouncer-mixin/targets.libsonnet [hunk 3]
```diff
@@ -131,5 +131,35 @@ local lokiQuery = g.query.loki;
       )
       + prometheusQuery.withLegendFormat('%s' % utils.labelsToPanelLegend(this.config.legendLabels)),
 
+    topDatabaseActiveConnection:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'topk by(database, instance, pgbouncer_cluster)(5, pgbouncer_pools_client_active_connections{%(clusterQuerySelector)s})' % vars
```
Comment (StefanKurek) [inline pgbouncer-mixin/targets.libsonnet:137]: Shouldn't we have a variable/selector for the Top K value?

**Review by StefanKurek — CHANGES_REQUESTED** [review]

**Review by JonathanWamsley — APPROVED** [review]
LGTM

**Review by StefanKurek — APPROVED** [review]

**Pull request merged**